### PR TITLE
IIS docs improvements

### PR DESCRIPTION
**Pull request by pdelewski**
Some improvements after recent ASP.NET IIS instrumentation testing.

### CODE REVIEW

**Review by nrcventura — APPROVED** [review]
Looks good to me, and the additional details seem useful.

**Review by RassK — APPROVED** [review]
Thanks for the input!

File: docs/iis-instrumentation.md [hunk 1]
```diff
@@ -62,6 +70,20 @@ to set it for all ASP.NET application running in Integrated Pipeline Mode:
   </location>
 ```
 
+> **Note** After applying above changes you can check whether `opentelemetry modules`
+> are loaded by using `appcmd` command.
+> Following example shows invocation for `WebDemo\` application:
+
+```terminal
+  appcmd list modules /app.name:"WebDemo/"
```
Comment (Kronos11) [inline docs/iis-instrumentation.md:78]: Maybe make it clear that this appcmd is found under %SystemDrive%\Windows\system32\inetsrv ?

File: docs/iis-instrumentation.md [hunk 2]
```diff
@@ -38,6 +41,11 @@ You can add it in the following places:
   </system.web>
 ```
 
+> **Warning**
```
Comment (Kronos11) [inline docs/iis-instrumentation.md:44]: good call out here

**Review by Kronos11 — APPROVED** [review]
Good additions

**Review by rajkumar-rangaraj — APPROVED** [review]

**Review by pjanotti — APPROVED** [review]

File: docs/iis-instrumentation.md (hunk 1 above)
Comment (pjanotti) [reply docs/iis-instrumentation.md:78]: Sorry, I merged before this could be addressed.

**Pull request merged**